Added social media icons in header

ankurg22 opened the pull request:
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showing after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- Social medias like Facebook, Twitter, Github, Youtube are added in the
top right corner of the header.
- https://ankurg22.github.io/gci17.fossasia.org/
- Closes #961

abishekvashok commented on _includes/header.html:
```diff
@@ -88,7 +88,27 @@
 								</ul>
-							</li>	
+							</li>
+              <li>
```
Can you please intend the code? It will be great. Rest looks fine.

ankurg22 replied:
Sure. Will make changes :blush:

ankurg22 committed:
Added social media icons in header

Social medias like Facebook, Twitter, Github, Youtube are added in the
top right corner of header.

Closes #961

rupav reviewed (commented):
Please indent the code, rest LGTM

ankurg22 commented:
@abishekvashok @rupav Done. Please review again :blush:

ParthS007 commented on _includes/header.html:
```diff
@@ -17,7 +17,7 @@
 	<link href='https://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600' rel='stylesheet' type='text/css'>
 	<link rel="icon" href="favicon.ico" />
-
+	
```
Remove whitespace

ankurg22 replied:
Sure.

ParthS007 commented on _includes/header.html:
```diff
@@ -39,7 +39,7 @@
 								<a href="{{ baseurl }}./#welcome">Welcome</a>
 							</li>
-     						<li>
+							<li>
```
Remove whitespace

ankurg22 replied:
Why this one? The code is properly indented here.

rupav replied:
This is fine.

ankurg22 commented:
@parths007 Please review now.

rupav commented:
Good work @ankurg22

gabru-md commented:
@rupav please usee Squash and Commit.
This avoids an additional merge commit in the logs.
:+1:
I've already told this to you.
But please make sure about this

rupav commented:
@gabru-md but its showing 1 commit 🤔
